package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "./scripts/build.sh",
     "dev": "./scripts/dev.sh",
+    "dev-novel": "NODE_ENV=development webpack-dev-server --config dev/webpack.config.js",
```

Comment:
We could probably reuse this command from the `dev.sh` script.

Reply:
What do you mean? This line was extracted from `dev.sh`.

Reply:
I mean in the `dev.sh` we can do `yarn dev-novel` instead of having the same command in two different place.

Reply:
That's not a big improvement though. Let's iterate on that later.